gtimelog2tick.py

```diff
@@ -265,6 +273,8 @@ def parse_timelog(
         task, text, task_id = parse_entry_message(config, entry.message)
         worklog = WorkLog(entry, text, task, task_id)
         if worklog.hours > 0:
+            if worklog.task_id is None:
+                raise DataError(worklog.text)
```

Comment:
I see now, that we need to calculate the hours to handle the first entry (which has `worklog.hours == 0` but is only handled implicitly here).

Reply:
Yes this was also my problem: to be able to ignore entries with 0 minutes you have to parse the message to create a Worklog instance. Maybe the factoring is not great here: It could be an idea to move parsing the message into the Worklog class and do it when necessary.

Let me try this.

Reply:
I refactored it in https://github.com/minddistrict/gtimelog2tick/pull/11/commits/5c3d819c01066fa8a4ccb2109d3f9dd3cb924c79.